assets/app/styles/_core.less

```diff
@@ -645,4 +645,12 @@ select:invalid {
 
 pre.clipped {
   display: inline-block;
+}
+
+// For when you need formatting of whitespace but don't want all the other pre tag styles
+pre.lightweight {
```

Comment:
Does it work to simply set `white-space: pre;` on the element? Or possibly `pre-wrap`?